src/main/pg/pg_ids.h

```diff
@@ -152,7 +152,8 @@
 #define PG_MODE_ACTIVATION_CONFIG 553
 #define PG_DYN_NOTCH_CONFIG 554
 #define PG_RX_EXPRESSLRS_SPI_CONFIG 555
-#define PG_BETAFLIGHT_END 555
+#define PG_SCHEDULER_CONFIG 536
```

Comment:
Is it intentional? should be 556?

Reply:
Good spot!

Reply:
I had to look at that for a few seconds before I spotted it. Made me feel dyslexic for a moment. I'm sticking the numbers into a vertically aligned column to aid readability.